Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'jquery-rails'
 gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 
-gem 'rest-client', '~> 1.8' # pinned to 1.x release line as 2.x breaks rake was_thumbnail_service:run_thumbnail_monitor
+gem 'rest-client', '~> 2.0' # '~> 1.8' # pinned to 1.x release line as 2.x breaks rake was_thumbnail_service:run_thumbnail_monitor
```

Comment:
Remove this comment?  Did you verify the rake task works now?

Reply:
Comment was removed in #108 however I'm happy to consolidate these 2 small PRs if preferred.

I'm confirming now, I'm not seeing any errors related to that job - but seeing not found errors against `openwayback.example.edu` which may be expected as that's in the example config.